internal/common/db.go

```diff
@@ -105,6 +105,9 @@ type InfraEnv struct {
 
 	// Timestamp for expiration of the image
 	ImageExpiresAt strfmt.DateTime `json:"image_expires_at" gorm:"type:timestamp with time zone"`
+
+	// Hosts that are associated with this infraEnv.
+	Hosts []*Host `json:"hosts" gorm:"foreignkey:InfraEnvID;association_foreignkey:ID"`
```

Comment:
why do we need Hosts in InfraEnv? I see that you are not using this info anywhere ( for now), and when you do need hosts you are using GetHostsFromDBWhere.

Reply:
removing it